src/metrics/clients.go

```diff
@@ -0,0 +1,53 @@
+package metrics
+
+import (
+	//"strconv"
+)
+
+type Version struct {
+	Version string
+	Count   int //needed?
```

Comment:
I get the idea of using a different struct for Clients-Versions-Counter to avoid iterating through the new `PeerStore`, however, if a peer updates the version it still would appear in the list of the previous one. Perhaps we should implement something like a tracker for when a peer updates it's version/client

Reply:
That shouldn't happen as I create a new `client` every time I export the metrics: see [this](https://github.com/migalabs/armiarma/pull/15/files#diff-3dac722eba627d5275c7abaa5c78b14b75b0b3fb4202316930626c415019368bR48).

Reply:
I see, closing the conversation.